### ISSUE TITLE
Citizen Engagement page: Use `image` instead of `card-image-src`

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#4159]
### Dependency
- #3756

### Overview
We need to refactor the code on the Citizen Engagement page to use `image` instead of `card-image-src` so that we aren't duplicating information.

### Details
- The `image` and `card-image-src` are fields in the project's Markdown files in `_project` directory. (Note: Not all of the project Markdown files have `card-image-src` yet.) 
- The `image` and `card-image-src` fields both have the path to the project's logo image. Since this information is the same, we don't need both fields. So, we will be removing the `card-image-src` fields from the Markdown files in `_projects` directory.
- Also, the alt text for each project logo on the Citizen Engagement page is empty (meaning `alt=""`) because the project name is displayed next to the logo. So, we will be removing the `card-image-alt` fields from the Markdown files in `_projects` directory.

### Action Items
- [x] In `_includes/program-area-pages-cards.html`, do the following:
  - [x] change
    ```
    <img class="card-image" src="{{ item.card-image-src }}" alt="{{ item.card-image-alt }}"/>
    ```
    to
    ```
    <img class="card-image" src="{{ item.image }}" alt="">
    ```
  - [x] In the if statement, remove the line: 
    ```
    item.card-image-src.size > 0 and
    ```
- [x] Remove the `card-image-src:` and `card-image-alt:` fields and the content of each of those fields in the following files:
  - [x] _projects/311-data.md
  - [x] _projects/access-the-data.md
  - [x] _projects/engage.md
  - [x] _projects/lucky-parking.md
  - [x] _projects/open-community-survey.md
- [x] Using the search function in VS Code or IDE, search for `card-image-src` and `card-image-alt`. If you find any files with that, update the code in those files accordingly.
- [x] Using Docker, check the Citizen Engagement page looks good in mobile, tablet, and desktop views.

#### Merge Team, Technical Lead, or Product Manager
- [ ] Once the pull request associated with this issue is merged, go to issue #3328 and check if all the issues listed under the `Dependency` section are closed. If they are all closed, move that issue (#3328) into the `New Issue Approval` column, remove the `Dependency` label on it, and add a `ready for dev lead` label to it. 

### Resources/Instructions
- Citizen Engagement page: https://www.hackforla.org/citizen-engagement
- [_includes/program-area-pages-cards.html](https://github.com/hackforla/website/blob/gh-pages/_includes/program-area-pages-cards.html)
- [Liquid template language](https://shopify.github.io/liquid/)
- This issue is tracked in #4153

**Comment by JessicaLucindaCheng** [comment]
Once this issue is in the New Issue Approval column, a dev lead needs to review the issue and determine if it is ready for a PM to prioritize it. If it isn't ready, a dev lead will need to edit this issue so that it is ready for a PM to prioritize it.

**Comment by ExperimentsInHonesty** [comment]
@Adaakal, if you want to keep this issue, you need to move it to in progress column and give your 
i. Availability: (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. ETA: (When do you expect this issue to be completed?)

**Comment by ExperimentsInHonesty** [comment]
@Adaakal I am unassigning you from this issue because you have already done a small issue.  I will let you know as soon as there is a medium size issue for you to do.  You are at the top of the list.

**Comment by drakenguyen4000** [comment]
i. Availability:  M-F 6-9pm 
ii. ETA:  3/30/23
